ZupIT/horusec
# Fixing migrate script

use docker run migrate container to execute the migration



**- What I did**

Update migrate script to execute the migration inside a docker container

**- How to verify it**

Running the make install command and verifying if the migrate script was successfully executed 

**- Description for the changelog**
